### ISSUE TITLE
Project support not released

### ISSUE THREAD

**Issue by scy** [koalalorenzo/python-digitalocean#318]
Hi there!

I’ve just tried to use the support for managing DO projects as developed in #300. Turns out, it’s not yet released, even though #290 was closed with “got merged and published”. 🙈

My guess would be that #300 was merged into `develop` by mistake and it should have been into `master`, or maybe `develop` into `master` afterwards, but neither was done. Looks like an oversight to me, especially because 1.15.0 was released on the same day as the merge (Jan 20, 5 months ago), but doesn’t include projects support.

The changes from #300 no longer apply without conflicts due to the addition of firewall code, but shouldn’t be too hard to fix. Should I open a pull request to merge an updated `develop` with fixed merge conflicts into `master`?

**Comment by scy** [comment]
Since I have a client that wants to use this feature, I’ve forked the repo, updated `develop` from `master` and polished the whitespace and docs a bit. #319 is my pull request.

**Comment by scy** [comment]
Closing since #319 has been merged.

**Comment by scy** [comment]
Oops. Yes, there’s been a merge, but no new release yet. Reopening. Sorry for the noise.

**Comment by tgardiner** [comment]
Just adding my support for this, waiting on a release that includes the project capability.  Looks like no release since Jan this year.

**Comment by koalalorenzo** [comment]
#325 fixed this, sadly with Micosoft's GitHub Actions :( 
This is a little more complicated than moving to GitLab but I have the feeling that nobody would contribute there :(
I will make a release soon and close this ;) Sorry for the delay!

**Comment by koalalorenzo** [comment]
Closing! I just made a new release: v1.16.0 :)
Again, I am so sorry for the delay! Please accept this cat picture 😍😜😅

![IMG_1587](https://user-images.githubusercontent.com/33528/99720686-60b3bc00-2aae-11eb-924d-737f3cd77601.JPG)
